.travis.yml

```diff
@@ -49,7 +49,7 @@ before_script:
  - git submodule update --init rsx_program_decompiler asmjit 3rdparty/ffmpeg 3rdparty/pugixml 3rdparty/GSL 3rdparty/libpng Utilities/yaml-cpp 3rdparty/cereal
  - mkdir build
  - cd build
- - if [ "$TRAVIS_OS_NAME" = "linux" ]; then cmake ..; else cmake .. -DLLVM_DIR=/usr/local/opt/llvm38/lib/llvm-3.8/share/llvm/cmake; fi
+ - if [ "$TRAVIS_OS_NAME" = "linux" ]; then cmake .. -DLLVM_DIR=/usr/share/llvm-3.8/cmake; else cmake .. -DLLVM_DIR=/usr/local/opt/llvm38/lib/llvm-3.8/share/llvm/cmake; fi
```

Comment:
You shouldn't need to specify LLVM location for linux, `find_package should `find it automatically (in the mac OS case it was required to use propper llvm and not apple-patched one).